### PR TITLE
Other fixes

### PR DESCRIPTION
**Pull request by GioSensation**
**Reviewer:** @shakyShane 
**Asana:** https://app.asana.com/0/1202427674957632/1202535734255841/f

## Description
A few other minor fixes.

- Yahoo Mail: https://app.asana.com/0/1202427674957632/1202535734255841/f
- Vancity: https://app.asana.com/0/1202427674957632/1202502625739236/f

## Steps to test
Automated tests added.

### CODE REVIEW

File: src/Form/Form.js [hunk 1]
```diff
@@ -392,8 +392,7 @@ class Form {
     shouldOpenTooltip (e, input) {
         if (this.device.globalConfig.isApp) return true
 
-        const inputType = getInputMainType(input)
-        return (!this.touched.has(input) && this.areAllInputsEmpty(inputType)) || isEventWithinDax(e, input)
+        return (!this.touched.has(input) && !input.classList.contains('ddg-autofilled')) || isEventWithinDax(e, input)
```
Comment (GioSensation) [inline src/Form/Form.js:395]: This was old logic for which `areAllInputsEmpty` was equivalent to the field having being autofilled, but now we have many more fields and we should check explicitly for `ddg-autofilled`.

File: integration-test/tests/login-form.ios.spec.js [hunk 2]
```diff
@@ -86,6 +86,27 @@ test.describe('Auto-fill a login form on iOS', () => {
...
+                    featureToggles: {
+                        inputType_credentials: true
+                    },
+                    availableInputTypes: {
+                        credentials: true
+                    },
+                    credentials
+                })
+                await login.promptWasShown('ios')
+                await page.pause()
```
Comment (shakyShane) [inline integration-test/tests/login-form.ios.spec.js:102]: oops, needs removing

File: integration-test/tests/login-form.ios.spec.js [hunk 3]
```diff
@@ -86,6 +86,27 @@ test.describe('Auto-fill a login form on iOS', () => {
...
+
+                await login.clickIntoPasswordInput()
+                const calls = await page.evaluate('window.__playwright.mocks.calls')
+                const mockCalls = calls.filter(([name]) => name === 'getAutofillData')
+                expect(mockCalls).toHaveLength(1)
```
Comment (shakyShane) [inline integration-test/tests/login-form.ios.spec.js:107]: Can we move these 3 lines into a method in pages? something like 

```javascript
await login.assertAutofillDataRequested()
```

I also think there's a helper for mock calls, something like: 

```javascript
const calls = await mockCalls(page, ["getAutofillData"]);
expect(calls).toHaveLength(1)
```

**Review by shakyShane — CHANGES_REQUESTED** [review]
This is great, just a couple of little bits :)

File: integration-test/tests/login-form.ios.spec.js (hunk 3 above)
Comment (GioSensation) [reply integration-test/tests/login-form.ios.spec.js:107]: The helper uses `find`, here we must check the exact number of calls (1 vs 2).

**Pull request merged**